modules/rosa-deleting-cluster.adoc

```diff
@@ -32,6 +46,39 @@ $ rosa init --delete-stack
 ----
 endif::sts[]
 
+ifdef::sts[]
+.  Delete the OpenID Connect (OIDC) provider that the cluster Operators use to authenticate:
++
+[source,terminal]
+----
+$ rosa delete oidc-provider -c <cluster_id> --mode auto --yes <1>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster.
+
+. Delete the cluster-specific Operator IAM roles:
++
+[source,terminal]
+----
+$ rosa delete operator-roles -c <cluster_id> --mode auto --yes <1>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster.
+
+. Delete the account-wide roles:
++
+[source,terminal]
+----
+$ rosa delete account-roles --prefix <prefix> --mode auto --yes <1>
+----
+<1> Replace `<prefix>` with the prefix of the account roles that are being deleted, for example `ManagedOpenShift`.
```

Comment:
Can we say a bit more detail?  Something like "If you defined a prefix when creating the account roles you must replace <prefix> here with that.  Otherwise the default is "ManagedOpenShift".

Reply:
I was thinking about this last week when I wrote this and subsequently had a conversation with @vkareh on this topic. 

My understanding is that for this command you must specify the `--prefix` option and include the value. This is even the case when you used the default `ManagedOpenShift` prefix when you created the roles. If you don't specify the prefix with this deletion command, even the default one, then it returns this error:

```
$ rosa delete account-roles 
Error: required flag(s) "prefix" not set
Usage:
[...]
```

Also, it can become a bit confusing if we say that the default is `ManagedOpenShift`, because that could imply that the deletion command will use that as a default value if not specified, which is not correct.

Based on this, I opted to minimise the wording around this while remaining unambiguous. I have also subsequently updated the command to state the following:

<1> You must include the `--prefix` argument. Replace `<prefix>` with the prefix of the account roles that are being deleted, for example `ManagedOpenShift`.

Reply:
I have updated the call out to the following, after a conversation with @0kashi:

> <1> You must include the `--prefix` argument. Replace `<prefix>` with the prefix of the account-wide roles that are being deleted. If you did not specify a custom prefix when you created the account-wide roles, the default `ManagedOpenShift` prefix would have been applied.